catalog/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ where verb is one of
 
 ## Changes
 
+- [Fixed] Docs: Fix `ui.package_description` example ([#4146](https://github.com/quiltdata/quilt/pull/4146))
```

Comment:
i don't think we need a catalog changelog entry for this

Reply:
Do we need a changelog entry somewhere else?

Reply:
Ideally, I would like to have this logged somewhere, because in my opinion, the change is significant.